Code-Hills/codehills-dev-profile-fe
# Ft profile

# User Profile Updates

## Description
This pull request adds the following user profile updates to the project:
- User can view their profile information, including name, email, and avatar.
- User can update their profile information.
- User can change their avatar by uploading a new image.

## Changes Made
The following changes were made to the codebase to support these updates:
- A new React component was created called `UserInformation` that renders the user's profile information and avatar.
- A new React component was created called `ProfileForm` that allows the user to update their name and email.
- A new React component was created called `ChangeAvatar` that allows the user to upload a new image to use as their avatar.

## Screenshots
![profile](https://user-images.githubusercontent.com/50295185/233449714-92287f25-592d-4d5e-b3fd-e14df187e9ea.png)
![edit-popup](https://user-images.githubusercontent.com/50295185/233449995-a30200b7-7d25-40f0-b702-3613ba1f200e.PNG)
![avatar](https://user-images.githubusercontent.com/50295185/233450938-76927550-5bf7-4e45-b421-70c562c51ae1.PNG)


## Testing
The following tests were added to ensure the new user profile functionality works as expected:
- checkout to `ft-profile` branch
- start app by `yarn dev`
- visit page in browser at: localhost:[port]/profile

## Known Issues
There are currently no known issues with this pull request.
